<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SBC Run Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/sbc_icon.png</normaloff>:/icons/sbc_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,9" columnminimumwidth="80,0">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="start_run_but">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:enabled {color: rgb(0, 85, 0);}
QPushButton:disabled {color: rgb(120, 120, 120);}</string>
      </property>
      <property name="text">
       <string>Start Run</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="run_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Run Num</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="run_state_label">
      <property name="text">
       <string>RunState</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="event_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Event Num</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="run_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="stop_run_but">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:enabled {color: rgb(170, 0, 0);}
QPushButton:disabled {color: rgb(120, 120, 120);}</string>
      </property>
      <property name="text">
       <string>Stop Run</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="event_num_edit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="event_time_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Event Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="event_time_edit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="run_live_time_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Run Live Time</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLineEdit" name="run_live_time_edit"/>
    </item>
    <item row="0" column="1" rowspan="11">
     <widget class="QTabWidget" name="tabs_widget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="plc_tab">
       <attribute name="title">
        <string>PLC</string>
       </attribute>
       <widget class="QToolButton" name="file_but">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>21</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
        </property>
       </widget>
       <widget class="QLineEdit" name="file_path">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>371</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="scint_tab">
       <attribute name="title">
        <string>Scintillation</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="acoustics_tab">
       <attribute name="title">
        <string>Acoustics</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="camera_tab">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>405</width>
             <height>331</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>1000</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="0" column="2">
             <widget class="QLabel" name="cam1_status_light">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(150, 150, 150);</string>
              </property>
              <property name="text">
               <string>Cam 1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="take_image_but">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Take Image</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="cam2_status_light">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(150, 150, 150);</string>
              </property>
              <property name="text">
               <string>Cam 2</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="cam3_status_light">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(150, 150, 150);</string>
              </property>
              <property name="text">
               <string>Cam 3</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="camera_status_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Camera Status:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_quit"/>
   <addaction name="action_settings"/>
   <addaction name="action_log"/>
  </widget>
  <action name="action_settings">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="action_log">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/log.png</normaloff>:/icons/log.png</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="toolTip">
    <string>Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stop_run_but</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stop_run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_settings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_settings_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_log</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_log_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_run_but</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_but</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>-10</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_run()</slot>
  <slot>quit()</slot>
  <slot>stop_run()</slot>
  <slot>open_settings_window()</slot>
  <slot>open_log_window()</slot>
  <slot>select_file()</slot>
 </slots>
</ui>
